infrastructure/resource_group_level/modules/cosmosdb.bicep

```diff
@@ -130,390 +130,368 @@ resource submissionsDb 'Microsoft.DocumentDB/databaseAccounts/mongodbDatabases@2
   }
 }
 
-resource banksCollection 'Microsoft.DocumentDB/databaseAccounts/mongodbDatabases/collections@2023-04-15' = {
```

Comment:
I've basically put all the resource definitions in an array (stripping out the `parent` as it's always the same.